### ISSUE TITLE
Timeout waiting for kernel_info reply

### ISSUE THREAD

**Issue by robmarkcole** [adafruit/circuitpython_kernel#15]
* CircuitPython Kernel version: 0.3.1
* Python version: Circuitpython 3.0.3 on Metro M0, system Python 3.6.5 |Anaconda, Inc.| (default, Apr 26 2018, 08:42:37)
* Operating System: MacOS 10.13.6

### Description

Getting the following timeout on starting the notebook:
```
[I 08:51:21.622 NotebookApp] Kernel started: b851b856-352e-4dc3-8265-d57d19b68915
DEBUG:asyncio:Using selector: KqueueSelector
DEBUG:asyncio:Using selector: KqueueSelector
DEBUG:circuitpython_kernel.board:/dev/cu.Bluetooth-Incoming-Port
DEBUG:circuitpython_kernel.board:/dev/cu.RobinsiPhone6-Wirelessi
DEBUG:circuitpython_kernel.board:/dev/cu.Bluedio-BluetoothSerial
DEBUG:circuitpython_kernel.board:/dev/cu.usbmodem14141
DEBUG:circuitpython_kernel.board:CircuitPython Board Found at:
DEBUG:circuitpython_kernel.board:/dev/cu.usbmodem14141
DEBUG:circuitpython_kernel.board:* entering raw repl...
[W 08:51:31.680 NotebookApp] Timeout waiting for kernel_info reply from b851b856-352e-4dc3-8265-d57d19b68915
```

Note that I am NOT getting this issue with https://github.com/goatchurchprime/jupyter_micropython_kernel

**Comment by brentru** [comment]
@robmarkcole  I just released a new version (0.3.2) of the library with a few performance tweaks and do not hit this error. Do you still get this error?

**Comment by robmarkcole** [comment]
HI @brentru my boards have circuitpython 3.1.1 now and I don't get this issue any more. Many thanks
